src/components/standalone/users_objects/CreateOrEditHostSetDrawer.vue

```diff
@@ -76,7 +91,15 @@ const ipVersionOptions = ref([
 ])
 
 const recordOptionsButCurrent = computed(() => {
-  return props.recordOptions?.filter((option) => option.id !== props.currentHostSet?.id)
+  // Filter out objects from recordOptions based on the presence of an IP address with a hyphen in allObjects
+  const objectsWithHyphenIp = props.allObjects
+    .filter((obj) => obj.ipaddr.some((ip: string) => ip.includes('-')))
+    .map((obj) => obj.name)
```

Comment:
Why are we considering `obj.name` instead of `obj.id`? Using the ID should be safer. I guess wel'll need to change line 101 too: `.includes(option.id)` (let's test this if we go this way)